packages/admin-ui/src/__mock-backend__/stakerConfig.ts

```diff
@@ -212,7 +212,7 @@ export const stakerConfig: Pick<
               isSelected: true,
               avatarUrl: "",
               metadata: {
-                name: "besu.dnp.dappnode.eth",
+                name: "goerli-besu.dnp.dappnode.eth",
```

Comment:
For the rest of the clients, you do not put "goerli-" here

Reply:
I don't know if it is a mistake or not

Reply:
Thanks for the comment! That file is just for the mock backend so is not that important